Update `osu!stream` (#13125)
ppy/osu-wiki: wiki/osu!stream/en.md

![](img/Os-Devices.jpg "osu!stream in iOS devices")

> ### App Store method (suggested)

Reviewer: ```suggestion
### App Store method (recommended)
```

> *Note: Make sure the device can connect to the Internet (either by Wi-Fi or 3G/4G).*

Reviewer: ```suggestion
*Note: Make sure the device can connect to the Internet (either by Wi-Fi or mobile data).*
```

- Go to "App Store" from the iOS device or "Play Store" on Android.
- Search for osu!stream
- Press and install osu!stream
  - If prompted to type in any account details, please do so.
- osu!stream can be played after the download and installation is completed.
- It is recommended to read the [Adding Beatmaps](#adding-beatmaps) section too to get the latest free songs pack. The current download contains only 2 songs.

> ### USB method (iOS-only)

Author: Do you want to keep this? Downloading via the app store is very common today.

Reviewer: i don't see the problem with leaving this here for documentation purposes, but it might be worth saying that this method is very much outdated lol

```suggestion
### USB method (legacy, iOS-only)
```

*Notice: For this method, [iTunes](https://www.apple.com/itunes/) must be installed.*

- Go to [this link and download it](http://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8).
  - If requested for Apple ID, type in the Apple ID and the password.
- Plug-in the iOS device to the device on which osu!stream is installed.
- Transfer the osu!stream files using iTunes.
- Unplug the iOS device and osu!stream can be played.

## Adding Beatmaps

> *Note: Make sure the device can connect to the Internet (either by Wi-Fi or 3G/4G).*

Reviewer: ```suggestion
*Note: Make sure the device can connect to the Internet (either by Wi-Fi or mobile data).*
```

- Open osu!stream.
- Press the osu! symbol to go to the main menu (or tutorial if playing for the first time)
- Press the "Store" bar
  - Alternatively, head to "Play" and press "Download more songs!"
- Select a desired pack.

## What is the difference?

- Beatmaps are made from scratch.
- No account needed to download the song packs.

> - osu!stream can be integrated with a Twitter account on iOS.

Author: I don't have iOS, so I can't verify this.

> - High scores may be submitted anonymously under the name of "Guest" (requires an internet connection).

Author: If I understand correctly, you cannot submit any new scores. Was the online mode shutdown entirely? If so, is this still true?

Reviewer: you can still access the online rankings but there's a notice that says `Leaderboards have been frozen. Thanks for participating!`

![image](https://github.com/user-attachments/assets/fa0e1264-3ff2-432e-8ed7-93548065b401)

> - The circle timer is replaced and moved to the bottom with a small border line which is coloured depending on the accuracy of the play. \[Bottom(Long)\]

Author: What does "Bottom(Long)" mean? Is this needed?

Reviewer: yeah i'd say just delete that lol

Author: ```suggestion
- The circle timer is replaced and moved to the bottom with a small border line which is coloured depending on the accuracy of the play.
```

- **Average Timing** is given at the grade page.
- Old records and replays cannot be saved.

> - The default skin cannot be changed and mods are not available (yet).

Reviewer: ```suggestion
- The default skin cannot be changed and mods are not available.
```

might just as well remove the (yet) since this will practically never be added lol

![](img/Os-HC.jpg "Hold circle 1")

> ---

Author: Do you want to keep these for visual separation?

Reviewer: feels like this should be okay either way
